docs/provider-pkcs11.7.md

```diff
@@ -258,6 +258,34 @@ Examples:
 ```PKCS11_PROVIDER_DEBUG=file:/dev/stderr,level:2```
 
 
+USE IN OLDER APPLICATIONS (URIs in PEM files)
+=============================================
+
+It is strongly suggested to update applications to use the new
+OSSL_STORE API provided by OpenSSL 3.0 which accepts URIs to
+transparenly load keys from either files or any other supported
+mechanism including pkcs11 URIs.
```

Comment:
would prefer to use uppercase PKCS#11 here and also in other places as it is still an abbreviation and term, rather than a word:
```suggestion
mechanism including PKCS #11 URIs.
```